### PR TITLE
Add fuzz testing for faucet.yaml config file

### PR DESCRIPTION
**Pull request by mab68**
Fuzz testing using py-afl-fuzz.

### CODE REVIEW

File: fuzz-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+bitstring
```
Comment (gizmoguy) [inline fuzz-requirements.txt]: Protip you can use "-r filename.txt" to include another requirements file.

I suspect the top of this file could be replaced with "-r test-requirements.txt"

Comment (mab68) [reply fuzz-requirements.txt]: AFL seems to take the test cases (provided in tests/in) and alters them in ways to produce errors. The mass amount of smaller files gives better results performance wise over one bigger file.

**Comment by gizmoguy** [comment]
Great, thanks for the cleanup of fuzz-requirements.txt

Other thing I would suggest is moving the example configurations that seed the AFL fuzzer to a directory with a better name, perhaps "tests/fuzzer/config/"

**Comment by gizmoguy** [comment]
This should be rebased/squashed now to clean up some of those commits.

We also need some documentation on how to run the fuzzer added to Sphinx.

**Pull request merged**